pkg/api/customization/workload/workload.go

```diff
@@ -64,6 +67,9 @@ func (a ActionWrapper) ActionHandler(actionName string, action *types.Action, ap
 			return httperror.NewAPIError(httperror.InvalidAction, fmt.Sprintf("Pause deployment %s before resume", deployment.ID))
 		}
 		return updatePause(apiContext, false, deployment, "resume")
+	case "redeploy":
+		timestamp := time.Now().UTC().Format(protocol.ISO8601TimeFormat)
```

Comment:
Can we use a format in the standard library(https://golang.org/pkg/time/)
As the package name suggested `github.com/aws/aws-sdk-go/private/protocol` is meant to be used internally so we should not depend on it.